examples/nivisa_dmm_measurement/README.md

```diff
@@ -24,29 +24,35 @@ and an NI Instrument Simulator v2.0.
 - NI-VISA
 - Optional: NI Instrument Simulator software
 
-Note: there is no Python instrument driver for the NI Instrument Simulator, so
-this example directly performs low-level, device-specific commands and queries.
+Supported instrument models:
```

Comment:
"Supported instrument models" belongs in "Required Hardware" not "Required Driver Software"